scanner.js

```diff
@@ -1457,9 +1457,11 @@ Scanner.prototype.revert_txids = function (callback) {
   self.to_revert = _.uniq(self.to_revert)
   if (!self.to_revert.length) return callback()
   console.log('need to revert ' + self.to_revert.length + ' txs from mempool.')
-  var n_batch = 100
-  // async.whilst(function () { return self.to_revert.length },
-    // function (cb) {
+  var more = true
+  var limit = 100
+  var skip = 0
+  async.whilst(function () { return more },
```

Comment:
The while loop was removed on purpose.
The reason was that if we restart bitcoind, the reverting loop of sometimes 50,000 txs takes a lot of time. In this time, not only that the explorer is not syncing up (new blocks, new mempool txs), but also transactions that *are* in the mempool but not shown yet because of the bitcoind restart will be reverted even if they will be discover by the node before the end of the reverting loop.
If I remove the loop and limit it to a small batch each time, than the next batch will take place after the syncing is progressed a while and a more clearer view of the mempool achieved (and maybe some new blocks).